index.js

```diff
@@ -1,9 +1,14 @@
 import 'react-native-gesture-handler';
 import { AppRegistry } from 'react-native';
-import PushNotificationIOS from '@react-native-community/push-notification-ios';
-import PushNotification from "react-native-push-notification";
-import {name as appName} from './app.json';
+import { name as appName } from './app.json';
 import App from './App';
+import messaging from '@react-native-firebase/messaging';
+import { CustomerIO } from "customerio-reactnative";
 
+messaging().setBackgroundMessageHandler(async remoteMessage => {
```

Comment:
From reading [the docs on Android message priority](https://firebase.google.com/docs/cloud-messaging/android/message-priority), it sounds like the `rnfirebase` docs might be misleading. 

From reading [the priority docs](https://firebase.google.com/docs/cloud-messaging/android/message-priority), it seems like data-only messages with normal priority will have the push notification *delayed* being sent to your app. So, the app will still receive your push to process, but it could have a delay. 

When you read the `rnfirebase` docs, however, it suggests...

> both Android & iOS treat the message as low priority and will ignore it (i.e. no event will be sent).

*...will ignore it* - ignore is a strong word making me think that the message will *never* be sent to the app to process? But maybe they mean the message is *initially ignored* by the Android OS but will *eventually* (after a delay) get delivered to the app?

Reply:
As a separate question from ^^^, 

what is the purpose of this line of code then?

Reply:
And just wanted to add, it sometimes do delivers with delay. But not every time, so we'll recommend customers with implementing both these handlers and use our rich push. If they report issues for notifications not delivered timely or being completely ignored, we'll suggest them to add `priority` as it mainly looks like limitation from `rnfirebase` and nothing from Firebase itself.

I've not yet completed docs PR for this, so I'll take a look if we can add some details for this to help customers.